hibernate-testing/src/main/java/org/hibernate/testing/junit4/CustomRunner.java

```diff
@@ -287,7 +291,7 @@ protected Ignore convertSkipToIgnore(FrameworkMethod frameworkMethod) {
 		);
 		if ( requiresDialectFeatureAnn != null ) {
 			try {
-				boolean foundMatch = false;
+				foundMatch = false;
 				for ( Class<? extends DialectCheck> checkClass : requiresDialectFeatureAnn.value() ) {
 					foundMatch = checkClass.newInstance().isMatch( dialect );
```

Comment:
We can simplify it to:

```
if ( !checkClass.newInstance().isMatch( dialect ) ) {
    return buildIgnore( requiresDialectFeatureAnn );
}
```

since `foundMatch` is no longer used outside of this loop.